### PR TITLE
 Refs [TASK][43216, 43217] Thiết kế giao diện danh sách tin tức  + responsive

### PR DESCRIPTION
**Pull request by khanh9a1999**
https://edu-redmine.sun-asterisk.vn/issues/43216
https://edu-redmine.sun-asterisk.vn/issues/43217

### CODE REVIEW

**Comment by khanh9a1999** [comment]
ready

File: pages/list-all-news.pug [hunk 1]
```diff
@@ -0,0 +1,6 @@
+extends layout/layout.pug
+
+include list-news/news-content
```
Comment (manhnt-0960) [inline pages/list-all-news.pug]: sao k include vào layout

Comment (khanh9a1999) [reply pages/list-all-news.pug]: ok rồi a

**Comment by khanh9a1999** [comment]
ready

**Pull request merged**